portfolio/src/main/webapp/index.html

```diff
@@ -46,23 +46,19 @@ <h3>Here are some places you can find me:</h3>
             <li><a href="https://github.com/EmmaWenzel">GitHub</a></li>
             <li><a href="https://www.linkedin.com/in/emma-wenzel-67b929155/">LinkedIn</a></li>
         </ul>
+      <h3>Click here to get a random fact:</h3>
+      <button onclick="addRandomFact()">Facts!</button>
+      <div id="fact-container"></div>
+      <br>
       <h3>
            This is a picture of a seagull. 
       </h3>
       <p> Seagulls are my second favorite
-           type of bird. This particular seagull is french; we'll call him
+           type of bird. This particular seagull is <i>french</i>; we'll call him
            Louis. I found Louis at Le Mont-Saint-Michel. He posed for my picutre.
         </p>
-      <img src="/images/IMG_5916.JPG" alt="seagull?" width = "300">
-
-      <p>This is my portfolio. Doesn't it look nice?</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
```

Comment:
seeing as addRandomGreeting() is no longer being used, has it been removed from the javascript files as well?

Reply:
Nope, but it's done now